src/protocols/ssh/settings.c

```diff
@@ -165,6 +166,12 @@ enum SSH_ARGS_IDX {
      */
     IDX_READ_ONLY,
 
+    /**
+     * Number of seconds between sending alive packets.  A default of 0
```

Comment:
The [documentation for `libssh2_keepalive_config()`](https://www.libssh2.org/libssh2_keepalive_config.html) defines some behavior which will make this not exactly correct:

> To avoid some busy-loop corner-cases, if you specify an interval of 1 it will be treated as 2.

That oddity will need to be documented. Perhaps we should just define 2 as the minimum value allowed?